extensions/amp-subscriptions/0.1/amp-subscriptions.css

```diff
@@ -75,3 +75,51 @@ html:not(.i-amphtml-subs-loggedin-yes) [subscriptions-action="logout"] {
 html:not(.i-amphtml-subs-subscriber-no) [subscriptions-action="subscribe"] {
   display: none !important;
 }
+
+
+/** Dialog. */
+amp-subscriptions-dialog {
+  display: block !important;
```

Comment:
isn't `width:100%` and `display:block` redundant? as `position:fixed` already makes element behave `display:inline`.

Or is this supposed to be `display:none` somewhere which this line overrides?

Reply:
No, `display:block` is still necessary, even if `position:fixed` defined. There are definitely nuances w/o specifying it.